bootstrap/selfconfig-root

```diff
@@ -71,6 +71,7 @@ my @required_debs = qw(
  libssl-dev
  nginx
  python3-certbot-nginx
+ rename
```

Comment:
Okay, but out of curiosity where is it used?

Reply:
It's used by journal-length-exporter, which is non-important monitoring, so isn't part of the normal bootstrap.  Maybe it shouldn't be here?

Reply:
I dropped it for now. It's not actually needed by pause.  (There is still an open question of how we want to track monitoring.)